samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -86,12 +85,18 @@ private enum SupportedFileType
             return this.BadRequest("File size exceeds the limit.");
         }
 
-        if (documentImportForm.DocumentScope == DocumentImportForm.DocumentScopes.Chat
-            && !(await this.UserHasAccessToChatAsync(documentImportForm.UserId, documentImportForm.ChatId)))
+        var chatSession = await this._chatSessionRepository.FindByIdAsync(documentImportForm.ChatId.ToString());
```

Comment:
The ChatId in the documentImportForm will only be valid when the scope is Chat. When the scope is Global, the document will be available to all chat. Please do not perform the validation here.

Reply:
Fixed to only do validation when the mode is chat.